### PR TITLE
relnote(117): Range is CORS-safelisted request header when a single bytes value is provided

### PR DESCRIPTION
**Pull request by bsmth**
The Range header is a cors-safelisted request header when there is a single range value. This is used for fetching media or resuming downloads and doesn't require sending preflight requests.

__Version numbers:__
  - __Fx:__ 117 https://bugzilla.mozilla.org/show_bug.cgi?id=1733981
  - __Chrome:__ 99: https://chromestatus.com/feature/5652396366626816
  - __Safari:__
    - 614.1.9, https://github.com/WebKit/WebKit/blob/cdb6817609f152fe3631c75a99ca5b1b702ec9c7/Source/WebCore/Configurations/Version.xcconfig#L24-L29
    - Commit changelog https://github.com/WebKit/WebKit/blob/cdb6817609f152fe3631c75a99ca5b1b702ec9c7/LayoutTests/imported/w3c/ChangeLog#L3-L12
    - 614.1.9 landed in version 16: https://github.com/mdn/browser-compat-data/blob/3cf58f25b614cda225d0ac33b3ee9cb9593a96b4/browsers/safari.json#L223-L228

__Docs:__

- [Range](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range) HTTP header reference
- [Range requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests) guide

__Related issues and pull requests:__
- [x] Parent issue https://github.com/mdn/content/issues/28281
- [x] Content https://github.com/mdn/content/pull/28733

__Additional resources:__
- WPT: https://wpt.fyi/results/cors/cors-safelisted-request-header.any.html?label=master&label=experimental&aligned
  - src: https://github.com/rayankans/wpt/blob/master/cors/cors-safelisted-request-header.any.js
- https://fetch.spec.whatwg.org/#concept-request-add-range-header

### CODE REVIEW

File: http/headers/Range.json [hunk 1]
```diff
@@ -35,6 +35,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cors_safe": {
+          "__compat": {
+            "description": "CORS-safelisted request header if single <code>bytes</code> range provided",
```
Comment (dipikabh) [inline http/headers/Range.json:41]: Could we make the desc shorter to:
"CORS-safelisted request header with single <code>bytes</code> range"

OR

"CORS-safelisted request header"

OR

"CORS-safelisted <code>Range</code> request header"

**Review by dipikabh — APPROVED** [review]
The updates look good, so leaving a +1. Feel free to merge after considering one small suggestion.

Thanks for including all the details in the PR description - they were pretty useful! 👍

File: http/headers/Range.json (hunk 1 above)
Comment (bsmth) [inline http/headers/Range.json:41]:
```suggestion
            "description": "CORS-safelisted request header with single <code>bytes</code> range",
```

Comment (bsmth) [reply http/headers/Range.json]: Thanks a lot:

>Could we make the desc shorter to:
"CORS-safelisted request header with single bytes range

Going with this one as the single `bytes` range is important to keep in, I think.

**Comment by bsmth** [comment]
Thank you, merging shortly 👍🏻

**Pull request merged**